<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormServer</class>
 <widget class="QWidget" name="FormServer">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>250</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>ServerConfig</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FormServer{background-color:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DFD7DF,stop:1 #8F8F8F); }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox_2{
background-color: rgba(255, 255, 255, 0);
border:2px solid white;border-radius:10px;
}</string>
     </property>
     <property name="title">
      <string>ServerConfig</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>服务器端IP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_internetip">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>192.168.1.6</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>服务器端PORT:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_internetport">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>50000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>服务器端身份验证码:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_internetcode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Mo5gkiU6Gypubag0mx8luQR9cvMvBNgu</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btn_internetselect">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#btn_internetselect{
background-color: rgba(170, 255, 255, 100);
border:2px solid white;border-radius:10px;
}

#btn_internetselect:hover{
background-color: rgba(170, 255, 255, 180);
border:2px solid white;border-radius:10px;
}


#btn_internetselect:pressed{
background-color: rgba(170, 255, 255, 255);
border:2px solid white;border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_internetselect_2">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#btn_internetselect_2{
background-color: rgba(170, 255, 255, 100);
border:2px solid white;border-radius:10px;
}

#btn_internetselect_2:hover{
background-color: rgba(170, 255, 255, 180);
border:2px solid white;border-radius:10px;
}


#btn_internetselect_2:pressed{
background-color: rgba(170, 255, 255, 255);
border:2px solid white;border-radius:10px;
}
</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
